src/apps/frontend/app.component.tsx

```diff
@@ -19,10 +19,12 @@ export default function App(): React.ReactElement {
   return (
     <AuthProvider>
       <AccountProvider>
-        <Toaster />
-        <Router>
-          <AppRoutes />
-        </Router>
+        <TaskProvider>
```

Comment:
Why does all the routes needs to be within TaskProvider, only Task specific routes should have this context.